### PR TITLE
[51] Fix Go Vet Makefile warnings

### LINKED ISSUE #51: Fix Go Vet Makefile warnings
When running go vet on govcd, there seems to be errors in types.go. Attached is the error:
screen shot 2018-08-23 at 8 18 34 am

### PR DESCRIPTION
**Pull request by Atul9**
closes #51 
Return result of fmt.Errorf call
Remove unnecessary space from struct tag value

### CODE REVIEW

File: govcd/edgegateway.go [hunk 1]
```diff
@@ -665,7 +665,7 @@ func (e *EdgeGateway) AddIpsecVPN(ipsecVPNConfig *types.EdgeGatewayServiceConfig
 
 	output, err := xml.MarshalIndent(ipsecVPNConfig, "  ", "    ")
 	if err != nil {
-		fmt.Errorf("error marshaling ipsecVPNConfig compose: %s", err)
+		return fmt.Errorf("error marshaling ipsecVPNConfig compose: %s", err)
```
Comment (auppunda) [inline govcd/edgegateway.go]: This will result in a compilation error. The function returns a Task and an error

**Comment by auppunda** [comment]
Thank you for your pull request, please fix the changes that I have outlined. We appreciate your contributions!!

File: govcd/edgegateway.go (hunk 1 above)
Comment (Atul9) [reply govcd/edgegateway.go]: @auppunda I have updated the PR

**Comment by auppunda** [comment]
LGTM. Thanks for helping make our project better!!

**Pull request merged**